corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -1,7 +1,5 @@
 from abc import ABCMeta, abstractmethod
 from sqlalchemy.exc import ProgrammingError
-from corehq.apps.es import GroupES, UserES
```

Comment:
I think you can move these back and use @proteusvacuum's suggestion for the tests